sopel/modules/reddit.py

```diff
@@ -337,41 +342,43 @@ def redditor_info(bot, trigger, match, commanded=False):
     bot.say(message)
 
 
-@url(user_url)
+@plugin.url(user_url)
+@plugin.output_prefix(PLUGIN_OUTPUT_PREFIX)
 def auto_redditor_info(bot, trigger, match):
-    redditor_info(bot, trigger, match.group(1))
+    return redditor_info(bot, trigger, match.group(1))
 
 
-@url(subreddit_url)
+@plugin.url(subreddit_url)
+@plugin.output_prefix(PLUGIN_OUTPUT_PREFIX)
 def auto_subreddit_info(bot, trigger, match):
-    subreddit_info(bot, trigger, match.group(1))
+    return subreddit_info(bot, trigger, match.group(1))
 
 
-@require_chanmsg('.setsfw is only permitted in channels')
-@commands('setsafeforwork', 'setsfw')
-@example('.setsfw true')
-@example('.setsfw false')
+@plugin.require_chanmsg('.setsfw is only permitted in channels')
```

Comment:
Sticky mess because it's not really possible to access the configured `help_prefix` at decoration time. Maybe easier to reword:

```suggestion
@plugin.require_chanmsg('Setting SFW status is only supported in a channel.')
```